lib/whitehall_importer/integrity_checker/body_check.rb

```diff
@@ -0,0 +1,29 @@
+module WhitehallImporter
+  class IntegrityChecker::BodyCheck
+    attr_reader :proposed_payload, :publishing_api_content
+
+    def initialize(proposed_payload, publishing_api_content)
+      @proposed_payload = proposed_payload
+      @publishing_api_content = publishing_api_content
+    end
+
+    def body_text_matches?
```

Comment:
As this class is focussed on body we don't really need to prefix this method with body_text

I think calling it something like "sufficently_similar?" similar might be enough to convey what it does.

Reply:
Changed name to `sufficiently_similar?`